kythe/go/indexer/emit.go

```diff
@@ -983,6 +984,29 @@ func (e *emitter) writeDoc(comments *ast.CommentGroup, target *spb.VName) {
 	e.writeFact(docNode, facts.NodeKind, nodes.Doc)
 	e.writeFact(docNode, facts.Text, strings.Join(lines, "\n"))
 	e.writeEdge(docNode, target, edges.Documents)
+	e.handleDeprecation(target, lines)
```

Comment:
Do we want the deprecation text to be escaped in the same way of the `doc` node text?  I don't think we need to render them with interpolation in the same way.

Reply:
So my thought here is "maybe yes" ... However neither the [Java](https://github.com/cushon/kythe/blob/0ab3f6a0011e8b060247a84ef7c451bfdf6c2bd2/kythe/java/com/google/devtools/kythe/analyzers/java/KytheDocTreeScanner.java#L158) nor [C++](https://github.com/cushon/kythe/blob/0ab3f6a0011e8b060247a84ef7c451bfdf6c2bd2/kythe/cxx/indexer/cxx/KytheGraphObserver.cc#L1041) do not. They should at least be consistent, so I'll do the Go without escaping.

Reply:
Done.